Products/CMFPlone/image_scales/indexer.py

```diff
@@ -0,0 +1,19 @@
+from Acquisition import aq_base
+from persistent.dict import PersistentDict
+from plone.dexterity.interfaces import IDexterityContent
+from plone.indexer.decorator import indexer
+from Products.CMFPlone.image_scales.interfaces import IImageScalesAdapter
+from zope.component import queryMultiAdapter
+from zope.globalrequest import getRequest
+
+
+@indexer(IDexterityContent)
+def image_scales(obj):
+    """
+    Indexer used to store in metadata the image scales of the object.
+    """
+    adapter = queryMultiAdapter((aq_base(obj), getRequest()), IImageScalesAdapter)
```

Comment:
Why aq_base here?

Reply:
i was thinking it was a good idea "isolate" the context, but probably not. removed it